extensions/roc-package-web-app-react/app/default/get-routes-and-store.js

```diff
@@ -56,15 +56,22 @@ export default function getRoutesAndStore() {
         store = storeCreator(replaceReducers);
     }
 
+    const { default: projectRoutes, middlewareConfig = {} } = require(REACT_ROUTER_ROUTES);
```

Comment:
General question. Right now we will blow up quite hard, unless some project routes have been defined. Maybe we should handle that a bit more gracefully?